apps/preferences/kinds/stars.cue

```diff
@@ -0,0 +1,27 @@
+package preferences
+
+starsV1alpha1: {
+  kind:       "Stars"
+  pluralName: "Stars"
+  scope:      "Namespaced"
+
+  validation: {
+    operations: [
+      "CREATE",
+      "UPDATE",
+    ]
+  }
+  schema: {
+    #Resource: {
```

Comment:
This shape will let us extend stars to other resources once we are in mode4+

Until then, dashboards is the only valid resource we can star